dsm/resource_aws_sobject.go

```diff
@@ -41,12 +41,13 @@ func resourceAWSSobject() *schema.Resource {
 		"**Note**: Once scheduled deletion is enabled, AWS security object can't be modified.\n\n" +
 		"**Deletion of a dsm_aws_sobject**: Unlike dsm_sobject, deletion of a dsm_aws_sobject is not normal.\n\n" +
 		"**Steps to delete a dsm_azure_sobject:**\n" +
-		"   * Enable schedule_deletion as shown in the examples of guides/dsm_azure_sobject.\n" +
-		"   * Enable delete_key_material as shown in the examples of guides/dsm_azure_sobject.\n" +
+		"   * Enable `delete_key_material` as shown in the examples of `Guides/dsm_aws_sobject`.\n" +
+		"   * Enable `schedule_deletion` as shown in the examples of `Guides/dsm_aws_sobject`.\n" +
 		"   * A dsm_aws_sobject can be deleted completely only when its state is `destroyed`.\n" +
-		"   * A dsm_aws_sobject is destroyed when the key is deleted from Azure key vault.\n" +
+		"   * A dsm_aws_sobject comes to destroyed state when the key is deleted from AWS KMS.\n" +
```

Comment:
=> "A dsm_aws_sobject's state is `destroyed` when the key is deleted from AWS KMS"

Reply:
Updated.